libs/layer-composer/src/generators/types.ts

```diff
@@ -263,6 +264,22 @@ export interface TrackGeneratorConfig extends GeneratorConfig {
   }
 }
 
+export interface PolygonsGeneratorConfig extends GeneratorConfig {
+  type: GeneratorType.Polygons
+  /**
+   * A GeoJSON made of one or more LineStrings. Features should have `coordinateProperties` set in order to filter by time
+   */
+  data: FeatureCollection
```

Comment:
It doesn't makes sense in this generator because it was an copy&paste issue but I've applied your suggestion on the [TrackGeneratorConfigData](https://github.com/GlobalFishingWatch/frontend/commit/8bbb2416fdfe47f688b740f1b5e407a7f1a410e6#diff-72bc4059b975f89c5bae06dc6e935ae0660f46ad894843cdc18b827101b60b5fR227) where it was copied from